app/services/foods_service.rb

```diff
@@ -0,0 +1,18 @@
+class FoodsService
+  def self.food_with(ingredient)
+    response = conn.get("/fdc/v1/foods/search?query=#{ingredient}&pageSize=10")
```

Comment:
nice job adding `pageSize=10` here so that you are limiting it to 10 results at the call